CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## unreleased
 
-* Bump braintree_android module dependency versions to `4.38.0`
+* Bump braintree_android module dependency versions to `4.38.1`
+* Upgrade `compileSdkVersion` and `targetSdkVersion` to API 34
```

Comment:
```suggestion
* Android 14 Support
  * Upgrade `compileSdkVersion` and `targetSdkVersion` to API 34
```
^ we got some merchant feedback about adding a note for Android 14 explicitly on the BT side so I YOLO pushed a commit there, but we should do the same here.